charts/umbrella/README.md

```diff
@@ -1,13 +1,22 @@
 - [Umbrella Chart](#umbrella-chart)
   - [Usage](#usage)
     - [Cluster setup](#cluster-setup)
+      - [Linux \& Mac](#linux--mac)
+      - [Windows](#windows)
     - [Network setup](#network-setup)
+      - [Linux \& Mac](#linux--mac-1)
+      - [Windows](#windows-1)
     - [Install](#install)
-      - [Released chart](#use-released-chart)
-      - [Repository](#use-local-repository)
+      - [Use released chart](#use-released-chart)
+        - [Option 1](#option-1)
+        - [Option 2](#option-2)
+      - [Use local repository](#use-local-repository)
+        - [Option 1](#option-1-1)
+        - [Option 2](#option-2-1)
     - [E2E Adopter Journeys](#e2e-adopter-journeys)
       - [Data exchange](#data-exchange)
       - [Get to know the Portal](#get-to-know-the-portal)
+      - [Note for onboarding process](#note-for-onboarding-process)
```

Comment:
should be a subsection of Get to know the Portal

Reply:
done